in_toto/models/layout.py

```diff
@@ -120,6 +120,27 @@ def read(data):
 
     return Layout(**data)
 
+  def display(self):
+    """
+    <Purpose>
+      Represents the Layout class by returning its fields in a readable string
+    <Returns>
+      Layout fields: type, expiration, readme, keyids, steps, inspections
```

Comment:
Please re-phrase. This contains a string and not individual fields.

Reply:
Sorry about that. Comments have been edited.